src/views/Player.vue

```diff
@@ -232,13 +258,26 @@ export default {
       // If first question, go to the start of the video
       // else go to the question which came just before the current ones
 
+      this.revised[currQuesIndex] += 1;
+
+      this.skipped[currQuesIndex] = 0;
+      this.submitted[currQuesIndex] = 0;
       // update response on S3
       this.uploadJson()
 
       this.player.currentTime = (currQuesIndex == 0) ? 0 : this.times[currQuesIndex - 1];
       this.player.play();
     },
 
+    listenToPlayButtons(){
+      (this.player.playing == true) ? 
+        this.played.push(this.player.currentTime) :
+        this.paused.push(this.player.currentTime)
+
+      this.uploadJson()
+      console.log(this.played)
+    },
+
```

Comment:
This function will be added to an eventListener -> that will listen to the clicks of the Play button (the big one and the small one). We want to record when the user played the video and when the user paused the video -> and this information will be pushed into respective arrays.